src/Nncase.Tests/Rewrite/Fusion/UnitTestFusionGroup.cs

```diff
@@ -56,15 +65,18 @@ private void RunCore(IDataFlowFusionCase fusionCase)
 
         IRModule module = new(main);
         CompilerServices.InferenceType(main);
+        Dumpper.DumpDotIR(main, "pre");
```

Comment:
Disable dump in Release mode for speedup CI.

Reply:
disabled